app/bundles/EmailBundle/EventListener/ReportSubscriber.php

```diff
@@ -324,11 +369,6 @@ public function onReportGenerate(ReportGeneratorEvent $event)
      */
     public function onReportGraphGenerate(ReportGraphEvent $event)
     {
-        // Context check, we only want to fire for Lead reports
-        if (!$event->checkContext('email.stats')) {
-            return;
-        }
-
```

Comment:
Are you sure that all the code except the one line bellow should be executed for all report types?